src/runtime_src/tools/scripts/xrtdeps-win22.py

```diff
@@ -67,6 +67,7 @@ def main():
   parser.add_argument('--icd', action="store_true", help='install Khronos OpenCL icd loader library')
```

Comment:
The copyright date should be updated (Line 2)

Reply:
Updated to 2023